README.md

```diff
@@ -1,7 +1,41 @@
 # CLA Public
 
+## Important notices
+
+- `master` branch is the default branch
+
+## Table of contents
```

Comment:
We will probably forget to update the table of contents on any subsequent update in the contents. (I would.) It is helpful, but perhaps we should consider moving some of these to a `docs/` folder or wiki?

Reply:
We could use something like https://ecotrust-canada.github.io/markdown-toc/ to generate manually.

If developers find a TOC helpful, we could think about automating this.

I think a new developer eyes will be handy to compare this README with READMEs in our other repositories.

Reply:
Sorry, what I meant was that the pattern of _needing_ a TOC means the readme is too large.

What do you think about separating each top-level section (dependencies, installation, development, testing, etc.) into separate files:

```
README.md
docs/
  dependencies.md
  installation.md
  ... etc
```

And adding links to them, ie. `[Dependencies](docs/dependencies.md)`?

This way we would have a structured readme but the cost of maintaining the structure is low and intuitive, without the need of external tools.

Reply:
Good idea!